source/PlayerInfo.cpp

```diff
@@ -577,6 +577,11 @@ void PlayerInfo::IncrementDate()
 	string message = accounts.Step(assets, Salaries());
 	if(!message.empty())
 		Messages::Add(message);
+	
+	// Advance the reload counters on all the player's weapons.
+	int advanceFrames = 24 * 60 * 60 * 60;	
```

Comment:
Might be right, I wasn't thinking about that. I'll do some reading / ask about, but yep, @endless-sky will be able to tell me in an instant. : )